.github/actions/daily_unittests/action.yml

```diff
@@ -0,0 +1,26 @@
+name: Run image
+description: Run private container from GitHub Packages Registry after Docker login
+inputs:
+  python_version:
+    description: 'Python version'
+    required: true
+  pandas_version:
+    description: 'Pandas version'
+    required: true
+  numpy_version:
+    description: 'Numpy version'
+    required: true
+  scikit_version:
+    description: 'Scikit-learn version'
+    required: true
+  tabmat_version:
+    description: 'Tabmat version'
+    required: true
+runs:
+  using: 'docker'
+  image: condaforge/mambaforge:latest
+  args:
+   - /bin/bash
+   - "-l"
+   - "-c"
+   - "./.github/workflows/test_daily.sh ${{ inputs.python_version }} ${{ inputs.pandas_version }} ${{ inputs.numpy_version }} ${{ inputs.scikit_version }}  ${{ inputs.tabmat_version }}"
```

Comment:
Please imitate our setup from the github actions repo and use environment variables instead of positional arguments.